### PR TITLE
Update keycloak_realm, add organizations_enabled

### PR DESCRIPTION
**Pull request by kominoshja**
##### SUMMARY
Add boolean parameter to enable organizations (still an experimental keycloak feature) feature toggling for keycloak

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
community.general.keycloak_realm

### CODE REVIEW

**Comment by russoz** [comment]
Hi @kominoshja 
Thank you for the contribution! Please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

**Comment by russoz** [comment]
Is the new option being consumed within the code directly from the module? I have not opened the whole file and won't be able to spend much time on it these days, but it looks like the new option is not used.

File: plugins/modules/keycloak_realm.py [hunk 1]
```diff
@@ -384,6 +384,12 @@
         aliases:
             - passwordPolicy
         type: str
+    organizations_enabled:
+        description:
+            - Enables support for experimental organization feature.
+        aliases:
+            - organizationsEnabled
+        type: bool
```
Comment (felixfontein) [inline plugins/modules/keycloak_realm.py:392]:
```suggestion
        type: bool
        version_added: 9.5.0
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by kominoshja** [comment]
Hey, sorry for the slow turnaround on my end, I've now added the changelog_fragment (Hope I got it right), and have also accepted @felixfontein's suggestion. 

Thank you all for the great work you do in maintaining these Ansible collections. With ❤️ 
Boris

**Comment by kominoshja** [comment]
> Is the new option being consumed within the code directly from the module? I have not opened the whole file and won't be able to spend much time on it these days, but it looks like the new option is not used.

It gets consumed by the code yes (organization_enabled is the key for this value in KC's api), so all I had to do was add it as another item in the meta_args dict

**Comment by kominoshja** [comment]
> > Is the new option being consumed within the code directly from the module? I have not opened the whole file and won't be able to spend much time on it these days, but it looks like the new option is not used.
> 
> It gets consumed by the code yes (organization_enabled is the key for this value in KC's api), so all I had to do was add it as another item in the meta_args dict

I also bumped added_version to 9.6.0 since .5 was already released

**Comment by russoz** [comment]
> I also bumped added_version to 9.6.0 since .5 was already released

9.5.0 is the last of its line and bears no heirs. Please bump it to 10.0.0 ;-)

(you can keep track of that at: #582 )

**Comment by felixfontein** [comment]
> 9.5.0 is the last of its line and bears no heirs. Please bump it to 10.0.0 ;-)

That's not completely true, there will be further 9.5.x bugfix releases, but no more 9.x.0 minor releases. So for new features it will be 10.0.0 (or later), but there will still be (bugfix release) heirs to 9.5.0 ;)

File: changelogs/fragments/9027-support-organizations-in-keycloak-realm.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - keycloak_realm - add boolean toggle to configure organization support for a given keycloak realm (https://github.com/ansible-collections/community.general/issues/9027).
```
Comment (felixfontein) [inline changelogs/fragments/9027-support-organizations-in-keycloak-realm.yml:2]:
```suggestion
  - keycloak_realm - add boolean toggle to configure organization support for a given keycloak realm (https://github.com/ansible-collections/community.general/issues/9027, https://github.com/ansible-collections/community.general/pull/8927/).
```

**Review by felixfontein — COMMENTED** [review]
One small change left from my side:

**Comment by russoz** [comment]
> > 9.5.0 is the last of its line and bears no heirs. Please bump it to 10.0.0 ;-)
> 
> That's not completely true, there will be further 9.5.x bugfix releases, but no more 9.x.0 minor releases. So for new features it will be 10.0.0 (or later), but there will still be (bugfix release) heirs to 9.5.0 ;)

C'mon I was trying to sound dramatic.

**Review by russoz — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@kominoshja thanks for your contribution!
@russoz thanks for reviewing!

**Comment by felixfontein** [comment]
FYI @kominoshja you might be interested in https://forum.ansible.com/t/keycloak-modules-in-community-general/41746

**Pull request merged**